requirements.txt

```diff
@@ -1,2 +1,3 @@
 mxnet==0.10.0
 numpy>=1.12
+tensorboard>=1.0.0a3
```

Comment:
I'm anyway not sure we want to add tensorboard as it is a optional dependency and is marked as that in the setup.py already. Do we need it for the Travis CI tests?